src/torrents/TorrentListItem.tsx

```diff
@@ -87,8 +97,8 @@ function TorrentListItem(props: Props) {
         }
       />
 
-      {props.secondaryAction ? (
-        <ListItemSecondaryAction>
+      {props.secondaryAction && !isXxs ? (
+        <ListItemSecondaryAction sx={{ display: ['none', 'block'] }}>
```

Comment:
This gets rid of whatever the secondary action would be.

However, you can still tap a torrent and then press start / stop in the top bar